frontend/src/scenes/onboarding/Onboarding.scss

```diff
@@ -0,0 +1,8 @@
+.ConfigGrid {
+    display: grid;
```

Comment:
I know we prefer not having scss files but I don't know how else to specify grid templates.

Reply:
https://tailwindcss.com/docs/grid-template-columns ;)